activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -523,6 +523,17 @@ def migration_connection # :nodoc:
       end
 
       private
+        def schema_cache_env
+          if ENV["SCHEMA_CACHE"]
+            ActiveRecord.deprecator.warn(<<~MSG.squish)
+              Setting `ENV["SCHEMA_CACHE"]` is deprecated and will be removed in Rails 7.3.
+              Configure the `:schema_cache_path` in the database configuration instead.
+            MSG
+
+            nil
```

Comment:
Shouldn't this return `ENV["SCHEMA_CACHE"]`? Deprecated doesn't mean it should have no effect. No?